Modules/Core/Common/include/itkMath.h

```diff
@@ -178,6 +179,10 @@ template <typename TReturn, typename TInput>
 inline TReturn
 Round(TInput x)
 {
+  static_assert(
+    sizeof(TReturn) > sizeof(int64_t) || std::is_integral_v<TReturn>,
+    "Math::Round template argument `TReturn` must be an integer type. If a floating point result is preferred, "
+    "consider using std::round (from <cmath>), or cast the result of a Math::Round call to a floating point type!");
```

Comment:
IMHO, it would be better to avoid suggestion of the _std::round_ ( _always_ rounds "to-nearest"), but itk::Math::Round actually rounds "upward". There are other functions, e.g. std::rint, std::nearbyint, they can be controlled via _fesetround_, but very often it has no effect, it depends on compiler flags related to floating point math, etc., so also bad. IMHO, just suggest to cast here should be safer.